src/Dev/DevelopmentAdmin.php

```diff
@@ -190,6 +176,33 @@ protected static function get_links()
         return $links;
     }
 
+    protected function getLinks(): array
+    {
+        $canViewAll = $this->canViewAll();
+        $links = [];
+        $reg = Config::inst()->get(static::class, 'registered_controllers');
+        foreach ($reg as $registeredController) {
+            if (isset($registeredController['links'])) {
+                if (!ClassInfo::exists($registeredController['controller'])) {
+                    continue;
+                }
+
+                if (!$canViewAll) {
+                    // Check access to controller
+                    $controllerSingleton = singleton($registeredController['controller']);
```

Comment:
```suggestion
                    $controllerSingleton = Injector::inst()->get($registeredController['controller']);
```
I was going to leave this alone but you've used this syntax in `TaskRunner` and I prefer it to the `singleton()` function.